package-system/AWSNativeSDK/FindAWSNativeSDK.cmake.Linux

```diff
@@ -121,6 +121,8 @@ ly_declare_aws_library(
         AccessManagement
     LIB_FILE 
         aws-cpp-sdk-access-management
+    BUILD_DEPENDENCIES
+        3rdParty::AWSNativeSDK::Core
```

Comment:
it may be worth a comment in one or two of these files explaining why we are explicitly declaring this build dependency here to resolve an issue with order so someone doesn't remove it in the future.